src/FEValues/FunctionValues.jl

```diff
@@ -107,6 +107,14 @@ function Base.copy(v::FunctionValues)
     d2Ndξ2_copy = _copy_or_nothing(v.d2Ndξ2)
     return FunctionValues(copy(v.ip), Nx_copy, Nξ_copy, dNdx_copy, dNdξ_copy, d2Ndx2_copy, d2Ndξ2_copy)
 end
+function task_local(v::FunctionValues)
+    Nξ = task_local(v.Nξ)
+    Nx = v.Nξ === v.Nx ? Nξ : task_local(v.Nx) # Preserve aliasing
```

Comment:
This should never happen

Reply:
What shouldn't happen?

Reply:
Nx should essentially never alias Nξ

Reply:
Yes, for identity mapping it does to save memory footprint

Reply:
But for identity mapped elements (like for example Lagrange) we still have Nx = J Nξ . For these Nx = Nξ iff J = I .

Reply:
Not according to https://ferrite-fem.github.io/Ferrite.jl/dev/topics/FEValues/#Identity-mapping.

Reply:
Oh, sorry, I mixed up the derivatives with the function values. Mb.